cypress/support/fragments/inventory/inventoryInstance.js

```diff
@@ -16,6 +31,10 @@ const deriveNewMarcBibRecord = Button({ id:'duplicate-instance-marc' });
 const addMarcHoldingRecordButton = Button({ id:'create-holdings-marc' });
 const viewHoldingsButton = Button('View holdings');
 const notesSection = Section({ id: 'instance-details-notes' });
+const moveItemsButton = Button({ id: 'move-instance-items' });
+const firstHolding = Accordion({ label: including('Holdings: Main Library') });
```

Comment:
need to organize expected hardcoded values. For example: move it into constants or objects or method parameters with meaning names